MonoGame.Framework/Input/Mouse.cs

```diff
@@ -140,9 +140,9 @@ public static MouseState GetState(GameWindow window)
 #elif DESKTOPGL || ANGLE
 
             var state = OpenTK.Input.Mouse.GetCursorState();
-            
-            window.MouseState.X = state.X;
-            window.MouseState.Y = state.Y;
+            var pc = ((OpenTKGameWindow)window).Window.PointToClient(new System.Drawing.Point(state.X, state.Y));
```

Comment:
I was testing a fix on Windows, Mac and Linux.

    var clientBounds = window.ClientBounds;
    window.MouseState.X = state.X - clientBounds.X;
    window.MouseState.Y = state.Y - clientBounds.Y;

Does the same thing, and cleaner.  Tested on Windows and Linux.  Couldn't get the project to build on Mac.  Some Xamarin Studio issue I think.